integration-test/src/test/java/com/datastrato/gravitino/integration/test/util/spark/SparkTableInfoChecker.java

```diff
@@ -78,6 +83,43 @@ public SparkTableInfoChecker withBucket(
     return this;
   }
 
+  public SparkTableInfoChecker withHour(List<String> partitionColumns) {
```

Comment:
How about `withHourPartition`? `withHour` seems too simple to understand.

Reply:
updated.